app/Http/routes.php

```diff
@@ -42,6 +42,7 @@
     Route::get('messages/{message}/send', 'MessagesController@sendMessage')->name('messages.send');
 
     // Users
+    Route::get('users/contestants', 'UsersController@contestantIndex')->name('users.contestants');
```

Comment:
was it 404-ing because of the `Route::get('users/{id}/signup/{signup_id}'` below?